### PR TITLE
Core6 Beta2

### PR DESCRIPTION
**Pull request by nhachicha**
- Also updated `next-major` against `origin/master`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,44 +13,49 @@ NOTE: This version bumps the Realm file format to version 10. It is not possible
...
 * RxJava Flowables and Observables are now subscribed to and unsubscribed to asynchronously on the thread holding the live Realm, instead of previously where this was done synchronously.
-* All RxJava Flowables and Observables now return frozen objects instead of live objects. This can be configured using `RealmConfiguration.Builder.rxFactory(new RealmObservableFactory(boolean))`. By using frozen objects, it is possible to send RealmObjects across threads, which means that all RxJava operators should now be supported without the need to copy Realm data into unmanaged objects.
+* All RxJava Flowables and Observables now return frozen objects instead of live objects. This can be configured using `RealmConfiguration.Builder.rxFactory(new RealmObservableFactory(true|false))`. By using frozen objects, it is possible to send RealmObjects across threads, which means that all RxJava operators should now be supported without the need to copy Realm data into unmanaged objects.
 * MIPS is not supported anymore.
 * Realm now requires `minSdkVersion` 16. Up from 9.
+* `IncompatibleSyncedFileException` is removed and no longer thrown.
```
Comment (cmelchior) [inline CHANGELOG.md]:
```suggestion
* [ObjectServer] `IncompatibleSyncedFileException` is removed and no longer thrown.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,44 +13,49 @@ NOTE: This version bumps the Realm file format to version 10. It is not possible
...
 * Added `RealmConfiguration.Builder.maxNumberOfActiveVersions(long number)`. Setting this will cause Realm to throw an `IllegalStateException` if too many versions of the Realm data are live at the same time. Having too many versions can dramatically increase the filesize of the Realm.
-* `RealmResults.asJSON()` is no longer `@Beta`.
-* Storing large binary blobs in Realm files no longer forces the file to be at least 8x the size of the largest blob.
-* Reduce the size of transaction logs stored inside the Realm file, reducing file size growth from large transactions.
+* `RealmResults.asJSON()` is no longer `@Beta`
 
 ### Compatibility
 * Realm Object Server: 3.23.1 or later.
-* File format: Generates Realms with format v10 (Reads and upgrades all previous formats from Realm Java 2.0 and later).
+* File format: Generates Realms with format v9 (Reads and upgrades all previous formats)
```
Comment (cmelchior) [inline CHANGELOG.md]: Was the previous entry a mistake?

Comment (nhachicha) [reply CHANGELOG.md]: No, just a merge mistake, I'll revert

Comment (nhachicha) [reply CHANGELOG.md]:
```suggestion
* File format: Generates Realms with format v10 (Reads and upgrades all previous formats from Realm Java 2.0 and later).
```

**Pull request merged**